web-app/src/routes/Home/Home.css

```diff
@@ -63,10 +63,11 @@
     text-align: center;
     height: 20px;
     min-width: 123px;
+    width: 123px;
```

Comment:
This looks like a bad conflict resolution. The width property was changed to min-width to fix an overflow issue. #168